web_src/js/features/repo-issue.js

```diff
@@ -87,21 +87,55 @@ export function initRepoIssueDue() {
 }
 
 export function initRepoIssueSidebarList() {
+  initIssueSearchDropdown('#new-dependency-drop-list');
+
+  function excludeLabel(item) {
+    const href = $(item).attr('href');
+    const id = $(item).data('label-id');
+
+    const regStr = `labels=((?:-?[0-9]+%2c)*)(${id})((?:%2c-?[0-9]+)*)&`;
+    const newStr = 'labels=$1-$2$3&';
+
+    window.location = href.replace(new RegExp(regStr), newStr);
+  }
+
+  $('.menu a.label-filter-item').each(function () {
+    $(this).on('click', function (e) {
+      if (e.altKey) {
+        e.preventDefault();
+        excludeLabel(this);
+      }
+    });
+  });
+
+  $('.menu .ui.dropdown.label-filter').on('keydown', (e) => {
```

Comment:
```suggestion
  $('.menu .ui.dropdown.label-filter').on('keyup', (e) => {
```

Usually it's user-friendlier to listen on keyup.